### DISCUSSION TITLE
Tactical RMM installation in local Network

### DISCUSSION THREAD

**Discussion by Gargi2021** [amidaware/tacticalrmm#2011] [Q&A] [answered]
Dear Friends,

Please guide me how to install Tactical RMM our local network

I have not any public IP can I installed with Local ip's

**Comment by dinger1986** [comment]
un --insecure with install, or you can still use it locally with a wildcard cert using the dns challenge, thats why its be done like that

**Reply by Gargi2021** [reply, marked as answer]
I have done by un --insecure , installation done but when I have access through IP address rmm site then not accessible , can you guide me how can setup wildcard cert or dns challange

**Reply by dinger1986** [reply]
Needs to use names not ips 

It's all in the docs
